packages/kbn-es-archiver/README.mdx

```diff
@@ -14,4 +15,42 @@ This tool also has a CLI which can be used to save indexes to new archives or lo
 
 To teach the ES Archiver how to talk to ES and Kibana it is ideal to start ES and Kibana using `node scripts/functional_test_servers --config some/config/file.ts` and then use the same `--config` flag when running `node scripts/es_archiver` so that it can access the location and authorization information about the ES instace from the FTR config file.
 
-Additional information about what functionality the CLI provides can be found by running `node scripts/es_archiver --help`
\ No newline at end of file
+Additional information about what functionality the CLI provides can be found by running `node scripts/es_archiver --help`
+
+## Performance Option Parameter
+
+We now have a performance parameter for the es-archiver#load(), entry-point 
+function.
+This parameter is optional, with defaults:
+
+ - HighWaterMark (stream batch size): 5000
```

Comment:
wouldn't it make sense to call this `'batchSize'` in the public interface? highWaterMark doesn't really have a trivial association with it (yet, I understand this is the name of the exact Node Stream parameter)

Reply:
I can work with that.  We all call it batch size too lol
Good one!